plankton/adapters/stream.py

```diff
@@ -152,6 +159,24 @@ def __init__(self, device, arguments=None):
 
         self._create_properties(self.commands)
 
+    @property
+    def documentation(self):
+        cmd_template = '{}:\n{}'
+
+        commands = [cmd_template.format(
+            cmd.pattern.pattern,
```

Comment:
Yes..I wanted to avoid storing the pattern and the compiled regex, the name `pattern` is arguably a bit...unfortunate :)